homeassistant/components/sensor/dte_energy_bridge.py

```diff
@@ -0,0 +1,82 @@
+"""Support for monitoring energy usage using the DTE energy bridge."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:flash'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the DTE energy bridge sensor."""
+    ip_address = config.get('ip')
+    if not ip_address:
+        _LOGGER.error(
+            "Configuration Error"
+            "'ip' of the DTE energy bridge is required")
+        return None
+    dev = []
+    dev.append(
+        DteEnergyBridgeSensor(ip_address,
+                              "Current Energy Usage", "instantaneousdemand",
```

Comment:
Out of curiosity, why are these passed as arguments if they are always the same?